app/workers/validator_score_v1_worker.rb

```diff
@@ -1,27 +0,0 @@
-# frozen_string_literal: true
```

Comment:
I removed this worker. Apart from that it had incorrect worker params structure (keywords), it also is not being used anywhere.